pkg/handlers/tdf.go

```diff
@@ -15,19 +16,29 @@ var (
 	ErrTDFInspectFailNotInspectable       = errors.New("file or input is not inspectable")
 	ErrTDFUnableToReadAttributes          = errors.New("unable to read attributes from TDF")
 	ErrTDFUnableToReadUnencryptedMetadata = errors.New("unable to read unencrypted metadata from TDF")
+	ErrTDFUnableToReadAssertions          = errors.New("unable to read assertions")
 	minBytesLength                        = 3
 )
 
-func (h Handler) EncryptBytes(b []byte, values []string, mimeType string, kasUrlPath string) (*bytes.Buffer, error) {
+func (h Handler) EncryptBytes(b []byte, values []string, mimeType string, kasUrlPath string, assertions string) (*bytes.Buffer, error) {
 	var encrypted []byte
 	enc := bytes.NewBuffer(encrypted)
 
+	var assertionConfigs []sdk.AssertionConfig
+	if assertions != "" {
+		err := json.Unmarshal([]byte(assertions), &assertionConfigs)
+		if err != nil {
+			return nil, errors.Join(ErrTDFUnableToReadAssertions, err)
+		}
+	}
+
 	// TODO: validate values are FQNs or return an error [https://github.com/opentdf/platform/issues/515]
 	_, err := h.sdk.CreateTDF(enc, bytes.NewReader(b),
 		sdk.WithDataAttributes(values...),
 		sdk.WithKasInformation(sdk.KASInfo{
 			URL: h.platformEndpoint + kasUrlPath,
 		}),
+		sdk.WithAssertions(assertionConfigs...),
```

Comment:
Should we leave the `sdk.WithAssertions` withOpt function off this `CreateTDF` call entirely if it is empty, or is passing it here with no values present alright?

Reply:
if there are no assertions, the slice will be empty and sdk will ignore processing the assertion configs.